docroot/modules/humsci/hs_siteimprove/hs_siteimprove.routing.yml

```diff
@@ -0,0 +1,17 @@
+hs_siteimprove.site_details:
+  path: '/admin/config/services/siteimprove/details'
+  defaults:
+    _controller: '\Drupal\hs_siteimprove\Controller\SiteImproveController::siteDetails'
+    _title: 'SiteImprove Site Details'
+  requirements:
+    _permission: 'access administration pages'
+
+hs_siteimprove.api_settings_form:
+  path: '/admin/config/services/siteimprove'
+  defaults:
+    _form: '\Drupal\hs_siteimprove\Form\SiteImproveSettingsForm'
+    _title: 'SiteImprove API Settings'
+  requirements:
+    _permission: 'access administration pages'
```

Comment:
I think we need a custom permission for this. There are roles that have basic access to the admin pages that probably shouldn't be able to change these settings.